runtimes/alibi-explain/mlserver_alibi_explain/runtime.py

```diff
@@ -46,6 +51,23 @@ def __init__(
         self.alibi_explain_settings = explainer_settings
         super().__init__(settings)
 
+    async def explain_v1_output(self, request: Request) -> Response:
```

Comment:
It should be possible to replace `Request` there for `InferenceRequest`, and then the decoding will happen automagically.

Reply:
done